bundles/framework/layerlist/view/LayerViewTabs/LayerList/LayerCollapse/Layer/LayerTools.jsx

```diff
@@ -15,6 +15,7 @@ const Tools = styled('div')`
     }
 `;
 
+//remove?
 const hasSubLayerMetadata = (layer) => {
```

Comment:
This seems to be unused, remove?

Reply:
Nice catch. Looks like it isn't exported and not used on the file itself so you can go ahead and remove this function.